docs/guides/stay-up-to-date/terraform/how-to-update-to-aws-provider-v4/core-concepts.md

```diff
@@ -0,0 +1,21 @@
+# Core Concepts
+
+Version [v4.0.0](https://github.com/terraform-providers/terraform-provider-aws/releases/tag/v4.0.0)
+of the Terraform AWS provider was released on February 10th 2022 with backward incompatible updates. Following the 
+provider release cycle, future releases from this point onward would only be compatible with 4.X. This means that if 
+you wish to use any new resources or data sources that are released from this point onwards, you will need to be 
+compatible with version 4 of the AWS provider to be able to pull those changes in.
+
+Originally v4 was a backward incompatible update, but since v4.9.0, released April 7th 2022, it has become backward
```

Comment:
Oh, I didn't realize that! Good to know.

Reply:
I know, right? It's eye-opening. Backward incompatible updates are so painful that it's better to do the work ahead of time to be as compatible as you can.